### PR TITLE
Add documentation for password recovery using email

### PR DESCRIPTION
**Pull request by vishesh92**
To be merged after: https://github.com/apache/cloudstack/pull/9509

----
📚 Documentation preview 📚: https://cloudstack-documentation--432.org.readthedocs.build/en/432/

### CODE REVIEW

**Comment by harikrishna-patnala** [comment]
@blueorangutan docbuild

File: source/adminguide/accounts.rst [hunk 1]
```diff
@@ -807,4 +807,30 @@ The admin can also disable 2FA for a User using the action button as shown below
           If the admin themself loses the authenticator application or forgets the static PIN, then the admin
           will have to either use apikey to disable 2FA using the API setupUserTwoFactorAuthentication with
           enable flag to false or to do the database changes in 'user' table by clearing the columns
-          'is_user_2fa_enabled', 'key_for_2fa', 'user_2fa_provider' for the specific entry.
\ No newline at end of file
+          'is_user_2fa_enabled', 'key_for_2fa', 'user_2fa_provider' for the specific entry.
+
+Setting up SMTP for Password Recovery
```
Comment (harikrishna-patnala) [inline source/adminguide/accounts.rst:812]:
```suggestion
Password Recovery for Users (Forgot Password)
```

Any other similar heading

File: source/adminguide/accounts.rst [hunk 2]
```diff
@@ -807,4 +807,30 @@ The admin can also disable 2FA for a User using the action button as shown below
...
+CloudStack supports password recovery using email. To enable this feature,
+set global setting `user.password.reset.enabled` to `true`. The following
+global settings are available to configure SMTP for password recovery.
+
+.. cssclass:: table-striped table-bordered table-hover
+
+================================================   ================   ===================================================================
+Global setting                                     Default values     Description
+================================================   ================   ===================================================================
+user.password.reset.enabled                        false              Determines whether password recovery via email is enabled or not
```
Comment (harikrishna-patnala) [inline source/adminguide/accounts.rst:824]: "user.password.reset.mail.template" setting is missing here, please check if anything else needs to be added.

File: source/adminguide/accounts.rst [hunk 3]
```diff
@@ -807,4 +807,30 @@ The admin can also disable 2FA for a User using the action button as shown below
...
           will have to either use apikey to disable 2FA using the API setupUserTwoFactorAuthentication with
           enable flag to false or to do the database changes in 'user' table by clearing the columns
-          'is_user_2fa_enabled', 'key_for_2fa', 'user_2fa_provider' for the specific entry.
\ No newline at end of file
+          'is_user_2fa_enabled', 'key_for_2fa', 'user_2fa_provider' for the specific entry.
+
+Setting up SMTP for Password Recovery
+--------------------------------------
+
+CloudStack supports password recovery using email. To enable this feature,
```
Comment (harikrishna-patnala) [inline source/adminguide/accounts.rst:815]: Please explain what happens when forgot password is entered and what user needs to be done. Attach screenshot of those.

**Comment by vishesh92** [comment]
@blueorangutan docbuild

**Review by harikrishna-patnala — APPROVED** [review]
LGTM

**Pull request merged**